### PR TITLE
Add SiteGround Optimizer cache invalidator

### PR DESCRIPTION
**Pull request by thunderdw**
This PR adds support for the [SiteGround Optimizer plugin](https://wordpress.org/plugins/sg-cachepress/) used by SiteGround.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,11 @@ wp acorn cachetags:database
 
 ## Usage
 
-Currently it only supports WP Super Cache but plan is to integrate with Cloudflare and other page caches.
+Currently it only supports WP Super Cache and SiteGround Optimizer but plan is to integrate with Cloudflare and other page caches.
+
+### SiteGround Optimizer
+
+Integration eists if you add the invalidator in the `config/cachetags.php` file.
```
Comment (oxyc) [inline README.md]:
```suggestion
Integration exists if you add the invalidator in the `config/cachetags.php` file.
```

File: src/Invalidators/SiteGroundCacheInvalidator [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<?php
```
Comment (oxyc) [inline src/Invalidators/SiteGroundCacheInvalidator]: Filename is missing `.php`

**Review by oxyc — COMMENTED** [review]
Thanks for the PR! I saw two minor change requests and then I can merge it

**Comment by thunderdw** [comment]
Great, thanks for the catches! Committed those changes

**Review by oxyc — APPROVED** [review]

**Comment by oxyc** [comment]
thanks!

**Pull request merged**